doc/source/cd/background.ipynb

```diff
@@ -0,0 +1,608 @@
+      "text/plain": [
+       "<Figure size 288x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Mean test accuracy 95.00%\n"
+     ]
+    }
+   ],
+   "source": [
+    "slope = true_slope\n",
+    "\n",
+    "X_0 = train_norm_0.rvs(size=60,random_state=2)\n",
+    "X_1 = train_norm_1.rvs(size=60,random_state=2)\n",
+    "X_test = np.vstack([X_0, X_1])\n",
+    "\n",
+    "# Plot\n",
+    "y_test = true_model(X_test,slope)\n",
+    "plot(X_test,y_test,slope,clf=clf)\n",
+    "\n",
+    "# Classifier accuracy\n",
+    "print('Mean test accuracy %.2f%%' %(100*clf.score(X_test,y_test)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Unsurprisingly, the model's mean test accuracy is relatively high. To run the detector on test data the `.predict()` is used:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'data': {'is_drift': 0,\n",
+       "  'distance': -0.004682348824606164,\n",
+       "  'p_val': 0.91,\n",
+       "  'threshold': 0.05,\n",
+       "  'distance_threshold': 0.010210391664106533},\n",
+       " 'meta': {'name': 'MMDDriftTF',\n",
+       "  'detector_type': 'offline',\n",
+       "  'data_type': None,\n",
+       "  'backend': 'tensorflow'}}"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "detector.predict(X_test)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As expected, since we are sampling from the distributions, the detector don't detect any covariate shift here. More specifically, the detector's p-value is above the threshold of $\\alpha=0.05$, indicating that no statistically significant drift has been detected."
```

Comment:
Typo: don't -> doesn't

Reply:
Mention that the distance (*S(x)* of the MMD detector) is below the threshold distance from the permutation test so no drift is flagged. It might be helpful to also add in a small paragraph on permutation testing.